ViacheslavBlazhkov/Lesson35/Middlewares/ThrowException.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Text;
+
+namespace Lesson35.Middlewares
+{
+    public class ThrowException
```

Comment:
What is purpose of this middleware? It doesn`t throw exception or handle it

Reply:
This middleware threw exception at start of program. Now I change it and middleware redirect us to other page if address contain /swagger and page where we want do go does not exist